app/models/financialDetails/ChargeItem.scala

```diff
@@ -200,6 +204,17 @@ object ChargeItem {
     getDisplayDueDate(charge2).isAfter(getDisplayDueDate(charge1))
   )
 
+  private val validChargeTypes = List(PoaOneDebit, PoaTwoDebit, PoaOneReconciliationDebit, PoaTwoReconciliationDebit,
+    BalancingCharge, LateSubmissionPenalty, FirstLatePaymentPenalty, SecondLatePaymentPenalty, MfaDebitCharge)
+
+  val validChargeTypeCondition: ChargeItem => Boolean = chargeItem => {
```

Comment:
might worth to add more context / change naming convention, as its not clear in what context charges are valid